simbad/score/dat_info.py

```diff
@@ -1,4 +1,13 @@
-class DatModelInfo(object):
+"""Class to store dat file info"""
+
+__author__ = "Adam Simpkin"
+__date__ = "16 April 2018"
+__version__ = "0.1"
+
+from simbad.score import ScoreBase
+
+
+class DatModelInfo(ScoreBase):
```

Comment:
Little detail but if we treat this as a score class, then we should call it `DatModelScore`, otherwise add a layer above the `ScoreBase` called `DataContainer`, which `ScoreBase` inherits and `DataBase`, then the respective models inherit from that? Or just use DataContainer for all of them ..., maybe rename the `score` subpackage to `datmodels` or something like this ... `score` seems really specific and ultimately will result in renaming

Reply:
Even better ... call it `core`

Reply:
I've altered this to `DatModelScore` for consistency. I think it's acceptable to call it a scoring class on reflection because although I'm using it to store data, I'm also using it to contain a difference score for the molecular weight.